client-src/elements/chromedash-app.js

```diff
@@ -445,6 +445,19 @@ class ChromedashApp extends LitElement {
         this.pageComponent.appTitle = this.appTitle;
       }
     );
+    page('/guide/intentpreview/:featureId(\\d+)', ctx => {
```

Comment:
This was actually used because we need to be able to show the intent template for devtrials (Ready for Developer Testing), but these stages do not have any gates associated with them. However, I've updated this to assume that passing "0" as the gate ID means that there is not a gate ID associated, and will show a relevant DevTrial intent.